web3_proxy/src/bin/web3_proxy_cli/migrate_stats_to_v2.rs

```diff
@@ -0,0 +1,345 @@
+                            Arc::new(request_metadata),
+                            (int_response_bytes)
+                                .try_into()
+                                .context("sum bytes average is not calculated properly")?,
+                        );
+                        // Modify the timestamps ..
+                        response_stat.modify_struct(
+                            int_response_millis,
+                            x.period_datetime.timestamp(), // I suppose timestamp is millis as well ... should check this in the (prod) database
+                            int_backend_requests,
+                        );
+                        // info!("Sending stats: {:?}", response_stat);
+                        stat_sender_ref
+                            // .send(response_stat.into())
+                            .send_async(response_stat.into())
+                            .await
+                            .context("stat_sender sending response_stat")?;
+                        // info!("Send! {:?}", stat_sender);
+                    } else {
+                        panic!("Stat sender was not spawned!");
+                    }
+
+                    // Create a new stats object
+                    // Add it to the StatBuffer
+                }
+
+                // Let the stat_sender spawn / flush ...
+                // spawned_app.app.stat_sender.aggregate_and_save_loop()
+                // Send a signal to save ...
+            }
+
+            // (3) Await that all items are properly processed
+            // TODO: Await all the background handles
+
+            // Only after this mark all the items as processed / completed
+
+            // If the items are in rpc_v2, delete the initial items from the database
+
+            // return Ok(());
+
+            // (4) Update the batch in the old table with the current timestamp (Mark the batch as migrated)
+            let old_record_ids = old_records.iter().map(|x| x.id);
+            let update_result: UpdateResult = rpc_accounting::Entity::update_many()
+                .col_expr(
+                    rpc_accounting::Column::Migrated,
+                    Expr::value(Value::ChronoDateTimeUtc(Some(Box::new(
+                        chrono::offset::Utc::now(),
+                    )))),
+                )
+                .filter(rpc_accounting::Column::Id.is_in(old_record_ids))
+                // .set(pear)
+                .exec(db_conn)
+                .await?;
+
+            info!("Update result is: {:?}", update_result);
+
+            // (N-1) Mark the batch as migrated
+            // break;
+        }
+
+        info!(
+            "Background handles (2) are: {:?}",
+            important_background_handles
+        );
+
+        // Drop the handle
+        // Send the shutdown signal here (?)
+        // important_background_handles.clear();
+
+        // Finally also send a shutdown signal
+
+        drop(stat_sender);
```

Comment:
as mentioned in another comment, i think this should be moved up higher

Reply:
see above comment: anything above will make it respawn at each batch as I understand, see https://github.com/yenicelik/web3-proxy/blob/stats_v3/web3_proxy/src/bin/web3_proxy_cli/migrate_stats_to_v2.rs#L274